docusaurus/docs/main-concept/generator.md

```diff
@@ -20,10 +20,15 @@ treats generate ./node_modules/@treats/addons-base/generator/opinionated
 ```
 There's several built-in generator that you can use:
 1. `component` - Generates React component boilerplates.
-2. `redux` -  Generates Redux boilerplates (action creator, reducer, thunks, etc).
-3. `test` - Generates Jest test boilerplates.
-4. `helper` -  Generates Treats helper object boilerplates.
-5. `middleware` -  Generates Treats middleware object boilerplates.
+2. `component-ts` - Generates React component boilerplates in Typescript.
```

Comment:
Instead of writing down each of these 5 features in [generator]-ts, how about changing into one line, saying that we also provide each of these rich features to support typescript, by typing the command we want to generate followed by `-ts`